### PR TITLE
Better YAML format, small issue fix

### PR DESCRIPTION
**Pull request by Tthomas63**
Format changes to competition yaml:
- [x] No `!!Python` tags
- [x] In order now

CompetitionDump view changes:
- [x] Delete button
    - [x] Check permissions
    - [x] Tests
    - [x] Make sure delete is done correctly

- [x] File size in mb

Small Issue:
    - Fix `Get Data` header appearing on every HTML content.

### CODE REVIEW

File: codalab/apps/web/tasks.py [hunk 1]
```diff
@@ -798,7 +798,7 @@ def do_phase_migrations():
 
 @task(queue='site-worker', soft_time_limit=60 * 60 * 24)
 def make_modified_bundle(competition_pk):
-
+    # The following lines help dump this in a nice format
     from collections import OrderedDict
```
Comment (ckcollab) [inline codalab/apps/web/tasks.py]: why is this import here?

File: codalab/apps/web/tasks.py [hunk 2]
```diff
@@ -807,7 +807,33 @@ def dict_representer(dumper, data):
 
     def dict_constructor(loader, node):
         return OrderedDict(loader.construct_pairs(node))
-
+    #Credit to miracle2k on Github for orderdict safe dump
```
Comment (ckcollab) [inline codalab/apps/web/tasks.py]: Space after #

File: codalab/apps/web/templates/web/competitions/_innertabs.html [hunk 3]
```diff
@@ -15,7 +15,9 @@
                     {% elif content.codename == "get_starting_kit" %}
                         {% include "web/competitions/_get_starting_kit.html" with competition=competition user=user %}
                     {% else %}
-                        <h1>Get data</h1>
+                        {% if content.codename == 'get_data' %}
+                            <h1>Get data</h1>
```
Comment (ckcollab) [inline codalab/apps/web/templates/web/competitions/_innertabs.html]: is this right? belongs here?

File: codalab/apps/web/views.py [hunk 4]
```diff
@@ -1859,3 +1859,21 @@ def start_make_bundle_task(request, competition_pk):
...
+class CompetitionDumpDeleteView(DeleteView):
+    model = models.CompetitionDump
+    template_name = 'web/competitions/delete_dump_confirm.html'
+
+    def dispatch(self, request, *args, **kwargs):
+        pk = kwargs.get('pk')
+        dump = models.CompetitionDump.objects.get(pk=pk)
+        competition = dump.competition
+        user = request.user
+        if user.id != competition.creator.id and user not in competition.admins.all():
```
Comment (ckcollab) [inline codalab/apps/web/views.py]: `user != competition.creator`

File: codalab/apps/web/templates/web/competitions/_innertabs.html (hunk 3 above)
Comment (Tthomas63) [inline codalab/apps/web/templates/web/competitions/_innertabs.html]: Yes. This is what fixes Get Data appearing everywhere.

**Comment by ckcollab** [comment]
🥘
